hook-common/src/webhook.rs

```diff
@@ -135,6 +137,11 @@ pub struct WebhookJobMetadata {
     pub team_id: u32,
     pub plugin_id: i32,
     pub plugin_config_id: i32,
+    #[serde(
+        serialize_with = "serialize_datetime",
+        deserialize_with = "deserialize_datetime"
+    )]
+    pub created_at: DateTime<Utc>,
```

Comment:
Are all fields of `WebhookJobMetadata` required? I'd argue that this one should be optional (and filled by api if not present)?

Reply:
They're required as currently written, yeah.

Reply:
Ok for now, but I'm not sure we should ask all sources to fill this. I'd be fine with making it an `Option<DateTime<Utc>>`

Reply:
Hm, the plugin-ness of the current code is expected, since the API contract very much matters to the janitor (for `app_metrics`). I guess the reason `Option` didn't come to mind here is that we only have 1 call-site that I've already updated.

Future non-plugin Webhook senders will require a little thought and probably their own endpoint, FWIW.

Reply:
Yep, let's keep it that way, but I wanted to callout that we're making a breaking change here.
Still fine for plugin-server only, we'll probably make another more generic API when opening to other use cases